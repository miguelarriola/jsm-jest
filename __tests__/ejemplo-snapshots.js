const cliente = {
  nombre: 'Juan Pol',
  balance: 500,
};

describe('Testing al cliente', () => {
  test('El es Juan Pablo', () => {
    expect(cliente).toMatchSnapshot();
  });
});

/* Actualizar snapdhot npm t -- -u */
