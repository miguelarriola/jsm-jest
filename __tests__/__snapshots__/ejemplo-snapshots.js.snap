// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al cliente El es Juan Pablo 1`] = `
Object {
  "balance": 500,
  "nombre": "Juan Pol",
}
`;
